### PR TITLE
[infinityprotocol-liquidity-pools] changes in infinityprotocol-liquidity-pools strategy

### PR DESCRIPTION
**Pull request by vatsalgupta13**
Fixes # .

Changes proposed in this pull request:
- edited infinityprotocol-liquidity-pools strategy so that any custom subGraphURL can be used
- Any project with Uniswap style subGraph can use this strategy to check balance of tokens in liquidity pools
- Added a scoreMultiplier so in case projects want to increase the scores of their liquidity providers

### CODE REVIEW

File: src/strategies/infinityprotocol-liquidity-pools/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+```JSON
+{
+  "strategies": [
+    ["infinityprotocol-liquidity-pools", {
+      // token parameters
+    "params": {
+        "address": "0xc168e40227e4ebd8c1cae80f7a55a4f0e6d66c97",
+        "symbol": "DFYN",
+        "decimal": 18,
```
Comment (ChaituVR) [inline src/strategies/infinityprotocol-liquidity-pools/README.md:18]: the decimal option is never used in the strategy file
```suggestion
        "symbol": "DFYN",
```

File: src/strategies/infinityprotocol-liquidity-pools/examples.json [hunk 2]
```diff
@@ -5,7 +5,9 @@
       "name": "infinityprotocol-liquidity-pools",
       "params": {
         "address": "0xd8a1734945b9ba38eb19a291b475e31f49e59877",
-        "symbol": "SHARD"
+        "symbol": "SHARD",
+        "decimal": 18,
```
Comment (ChaituVR) [inline src/strategies/infinityprotocol-liquidity-pools/examples.json:9]: the decimal option is never used in the strategy file
```suggestion
        "symbol": "SHARD",
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/infinityprotocol-liquidity-pools/README.md (hunk 1 above)
Comment (vatsalgupta13) [reply src/strategies/infinityprotocol-liquidity-pools/README.md]: You're right. Removed the decimal field.

File: src/strategies/infinityprotocol-liquidity-pools/examples.json [hunk 3]
```diff
@@ -6,6 +6,7 @@
       "params": {
         "address": "0xd8a1734945b9ba38eb19a291b475e31f49e59877",
         "symbol": "SHARD"
```
Comment (ChaituVR) [inline src/strategies/infinityprotocol-liquidity-pools/examples.json]:
```suggestion
        "symbol": "SHARD", 
```

Comment (vatsalgupta13) [reply src/strategies/infinityprotocol-liquidity-pools/examples.json]: Yeah realized that. Added now.

**Review by bonustrack — APPROVED** [review]

**Review by ChaituVR — APPROVED** [review]

**Comment by ChaituVR** [comment]
Strategy is updated: https://snapshot.org/#/strategy/infinityprotocol-liquidity-pools

**Pull request merged**